15_totalIntegers/solution/totalIntegers-solution.js

```diff
@@ -0,0 +1,15 @@
+const totalIntegers = function(obj, count = 0) {
+
+  if (typeof obj !== 'object' || obj === null) return;
```

Comment:
```suggestion
  if (typeof obj !== 'object' || obj === null) {
    return;
  }
```
For consistency with the braced conditions in the below loop.